lms/static/sass/views/_shoppingcart.scss

```diff
@@ -341,21 +196,22 @@
   }
 
   input[type="text"], input[type="email"], select {
-    font-family: "Open Sans",Verdana,Geneva,sans-serif,sans-serif;
-    font-style: normal;
-    border: 2px solid $dark-gray2;
+    @extend %t-icon5;
+    @extend %t-strong;
+    border: $input-border;
+    padding: ($baseline*0.4) ($baseline*0.6);
+    width: ($baseline*13);
     height: auto;
-    padding: 8px 12px;
-    font-weight: 600;
-    width: 260px;
-    font-size: 16px;
+    font-family: $sans-serif;
+    font-style: normal;
 
+    // STATE: focus
     &:focus {
-      border-color: $dark-gray2;
       box-shadow: none;
-      outline: none;
```

Comment:
a11y requires such visual indication in browsers (in case it's not provided by other means)